### DISCUSSION TITLE
hello sir. This program is not working

### DISCUSSION THREAD

**Discussion by Jo-NH** [RogueEdit/onlineRogueEditor#94] [Q&A] [answered]
I used number 2 but it doesn't work like this
![캡처](https://github.com/RogueEdit/onlineRogueEditor/assets/171997419/708b81d7-7011-4d97-bcdc-0020273a5f26)

**Comment by JulianStiebler** [comment, marked as answer]
Fixed as of now. Please use the issues feature next time :D
